### DISCUSSION TITLE
post_233920

### DISCUSSION THREAD

**Discussion by giscus** [cincuranet/blog-comments#1356] [Comments]
# post_233920

https://www.tabsoverspaces.com/233920-solving-logitech-brio-flickering-and-crashing-my-pc

**Comment by Reaper2507** [comment]
thanks a ton SAME ISSUE i was starting to want to do the SAME SHIT (already cables..) in the end 1 hr later i found your site!!! and i put it into my pc with a older 3.0 PCIE Card instead of my mainhub with 10gbs and SEEE.. SOLVED...

**Reply by Reaper2507** [reply]
ok i had no crashe's but powerpoint presentations just couldnd record and stopped in the middle..

**Reply by cincuranet** [reply]
Glad it worked.

**Comment by danbarker** [comment]
We have had the same issue of flickering with Logitech Brio webcams. I initially bought 2 original Brio Ultra (Business), and for both, the fix for flickering was to download the Logitune app and change the anti-flicker from NTSC to PAL. I subsequently bought 3 Brio Stream webcams. 1 of the 3 had no flicker, and 2 of them did. Changing the anti-flicker settings didn't work, so I sent them back, and bought another 2 original Brios (non-Stream). Of these, one had the flicker, and one didn't. So it seems like it's device-dependent - going to send this one back and try again!

**Comment by AsifMahsud** [comment]
Thank you so much for sharing this information! I struggled with this issue for a long time, thinking it was related to my graphics card. I tried different drivers and switched operating systems, but nothing worked. Your suggestion to change the USB port from 10G to 5G was amazing and resolved the problem!

**Comment by ralf-vogel** [comment]
I have noticed, right before the system freezes, there was a "The driver entered RTD3" log message in "Windows Logs -> System" by `nhi`. That's the "Native Host Interface", entering the low-power state (Runtime D3) - meaning that Windows tries to put the controller to sleep, while the cam is actively streaming.

So the solution is to:
1. Open the device manager
2. Find all the USB hubs
3. Open "Properties -> Power Management"
4. Disable the flag "Allow the computer to turn off this device to save power"

Since then, my Brio 500 didn't freeze my system any more. Hope that helps 😉
